src/js/components/leftPanel/layersPanel/DateRange.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState, ChangeEvent } from 'react';
+
+import { LayerProps } from 'js/store/mapview/types';
+
+interface DateRangeProps {
+  layer: LayerProps;
+}
+
+const DateRange = (props: DateRangeProps): JSX.Element => {
+  const { layer } = props;
+  console.log('<DateRange/>', layer);
+
+  const returnDateToday = (): string => {
+    const dateToday = new Date().toLocaleDateString();
+    const [monthToday, dayToday, yearToday] = dateToday.split('/');
+    const dayTodayFormatted = dayToday.length === 1 ? `0${dayToday}` : dayToday;
+    const monthTodayFormatted =
+      monthToday.length === 1 ? `0${monthToday}` : monthToday;
+    const dateTodayFormatted = `${yearToday}-${monthTodayFormatted}-${dayTodayFormatted}`;
+
+    return dateTodayFormatted;
+  };
```

Comment:
let's lift this helper function out of the `DateRange` component scope, just above it. No need to run this function every time we select a new date in the component?